docs/other/install-cvmfs-stratum1.md

```diff
@@ -0,0 +1,338 @@
+
+Install a CVMFS Stratum 1
+===================================================================
+
+# About this Document 
+This document describes how to install a CVMFS Stratum 1. There are many different variations on how to do that, but this document focuses on the configuration of the OSG GOC Stratum 1 oasis-replica.opensciencegrid.org. It is applicable to other Stratum 1s as well, very likely with modifications (some of which are suggested in the document below).
+
+# Applicable versions 
+The applicable software versions for this document are cvmfs and cvmfs-server >= 2.2.1.
+
+# Requirements
+
+## Host and OS
+
+-   OS is a 64-bit <span class="twiki-macro SUPPORTED_OS"></span>.
```

Comment:
Bad TWiki here, link to `release/supported_platforms` instead.

Reply:
Updated